src/qml/UnifiedPatchPage.qml

```diff
@@ -162,7 +162,7 @@ Page {
                 font.pixelSize: Theme.fontSizeSmall
                 textFormat: Text.StyledText
                 property string link: SOURCE_REPO + "/blob/master/README.md#for-developers"
-                text: qsTranslate("", "This patch uses the legacy format for its patch.json file. If you are the maintainer, do consider updating to the new format. If you are using Web Catalog you should not include a patch.json file at all in your upload.<br />See the developer section in the <a href=\"%1\">README</a> for more information.").arg(link)
+                text: qsTranslate("", "This patch uses the legacy format for its patch.json file. If you are its maintainer, please do consider updating to the new format; if you are using the Web Catalog you shall not include a patch.json file in your upload!&lt;br /&gt;See the developer section in the &lt;a href=&quot;%1&quot;&gt;README&lt;/a&gt; for details.").arg(link)
```

Comment:
Hey @Olf, please do not use XML escapes for these strings, they are supposed to contain valid HTML.

Reply:
Sorry, my bad: https://github.com/sailfishos-patches/patchmanager/pull/200#issuecomment-965614265